apps/src/templates/sectionAssessments/PercentAnsweredCell.story.jsx

```diff
@@ -1,22 +1,20 @@
 import React from 'react';
 import PercentAnsweredCell from './PercentAnsweredCell';
 
-export default storybook => {
-  storybook
-    .storiesOf('SectionAssessments/PercentAnsweredCell', module)
-    .addStoryTable([
-      {
-        name: 'Show Check Mark',
-        description:
-          'A green check mark is displayed when the answer is correct.',
-        story: () => (
-          <PercentAnsweredCell percentValue={40} isCorrectAnswer={true} />
-        )
-      },
-      {
-        name: 'Hide Check Mark',
-        description: 'Check mark is hidden when the answer is wrong.',
-        story: () => <PercentAnsweredCell percentValue={60} />
-      }
-    ]);
+export default {
+  title: 'PercentAnsweredCell',
+  component: PercentAnsweredCell
+};
+
+const Template = args => <PercentAnsweredCell {...args} />;
+
+export const ShowCheckMark = Template.bind({});
+ShowCheckMark.args = {
```

Comment:
Small nit here - "ShowCheckMark" describes the visual, but not the context - which is that the answer is correct. I wonder if it would makes sense to rename to something like "CorrectAnswer", so that the connection between "CheckMark" and "Correct" is clearer? To rule out the checkmark being something like "seen" or "graded". Open to discussion here!

Reply:
Good call- I updated this to CorrectAnswer!